website/docs/d/aws_static_credentials.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: "vault"
+page_title: "Vault: vault_aws_static_credentials data source"
+sidebar_current: "docs-vault-datasource-aws-static_credentials"
+description: |-
+  Reads static role credentials from an AWS secret backend in Vault
+---
+
+# vault\_aws\_static\_credentials
+
+Reads static role credentials from an AWS secret backend in Vault
+
+~> **Important** All data retrieved from Vault will be
+written in cleartext to state file generated by Terraform, will appear in
+the console output when Terraform runs, and may be included in plan files
+if secrets are interpolated into any resource attributes.
+Protect these artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+~> **Note**
+When using the outputs of this data source to authenticate with the [Terraform Provider for AWS](https://registry.terraform.io/providers/hashicorp/aws/latest/docs) or
+the [Terraform Provider for AWS Cloud Control](https://registry.terraform.io/providers/hashicorp/awscc/latest/docs),
+the credentials leased from Vault cannnot be renewed.
+Ensure that the lease is long enough for Terraform to complete.
+
+## Example Usage
+
+```hcl
+resource "vault_aws_secret_backend" "aws" {
+	path = "my-aws"
+	description = "Obtain AWS credentials."
```

Comment:
nit: The standard is 2 spaces for hcl files. I think we should try to be consistent in our docs.

Reply:
shoutout to the part of the test code that ended up as 3 space tabs somehow

Reply:
spaced!